pkgs/by-name/pi/picom-pijulius/package.nix

```
@@ -0,0 +1,27 @@
+{
+  picom,
+  lib,
+  fetchFromGitHub,
+  pcre
+}:
+picom.overrideAttrs (
+  finalAttrs: previousAttrs: {
+    pname = "picom-pijulius";
+    version = "2024-05-17-unstable";
```

My brain was afk as i was rechecking the version
 Looks like you missunderstood the version nameing sheme.

First should komme the last released version followed by -unstable- and finished with the date of the commit being refrenced in the format yyyy-mm-dd

```suggestion
    version = "8.2-unstable-2024-04-30";
```

---

Will do, thanks for the review